app-web/plugins/gatsby-source-github-all/__tests__/helpers.itest.js

```diff
@@ -0,0 +1,14 @@
+import { createPathWithDigest } from '../utils/helpers';
```

Comment:
A bit hard to read when everything is bunched up like this; a little spacing would be ok.